index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home :: First Capstone Project </title>
+    <!-- fontawesome css1 -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="./assets/images/icon.svg" rel="icon" type="img/svg" sizes="16x16"/>
+    <link rel="stylesheet" href="assets/css/home.css">
+</head>
+<body>
+    <header class="header">
+        <section class="top-nav">
+            <ul class="row container">
+                <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class="fa-brands fab fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </section>
+        <nav class="navbar">
+            <div class="mobile-overlay"></div>
+            <div class="mobile-foreground"></div>
+            <section class="container nav">
+                <div class="logo">
+                    <a href="./"><img src="assets/images/icon.svg" alt="logo" class="logo"></a>
+                </div>
+                <ul class="bars">
+                    <li class="bar-item"></li>
+                    <li class="bar-item"></li>
+                    <li class="bar-item"></li>
+                </ul>
+                <ul class="nav-link">
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="#speaker">Speakers</a></li>
+                    <li><a href="#contact">Sponsor</a></li>
+                    <li><a href="#contact">News</a></li>
+                    <li><a href="#" class="btn btn-outline-primary">Campaign</a></li>
+                </ul>
+            </section>
+        </nav>
+        <section class="hero container">
+            <p class="greeting">"Hello! Sharing world"</p>
+            <h1>MV FULL-STACK <br> GRADUATION EVENT <br> 2023 </h1>
```

Comment:
- [ ] Please try to reduce `line-height ` between your heading title to make it similar to the original design for both index and about pages :+1: